### DISCUSSION TITLE
HTTP1 vs HTTP2 vs HTTP3

### DISCUSSION THREAD

**Discussion by rizafahmi** [ngobrolin/web#35] [Ideas]
Referensi: https://blog.bytebytego.com/p/http1-vs-http2-vs-http3-a-deep-dive

**Comment by Jocerdikiawann** [comment]
Nah ini nih menarik, sekalian webtransport pak meski masih draft ietf saya pengen tau potensinya, karna mulai banyak yang bandingin webtransport sama websocket, jadi pembahasan menarik tuh.
